### ISSUE TITLE
CSV output in Python 3 mixes bytes and strings

### ISSUE THREAD

**Issue by QuLogic** [codeauroraforum/lid#2]
When using `-Fcsv` on Python 3, the resulting file is a mix of bytestrings and regular strings:
```
b'./LICENSE.md',Apache-2.0,0.9009905536934706,4,0,176,0,10143,1.0,"b'\r\n                                 Apache License\r\n ...'"
```

**Comment by craigez** [comment]
Thanks. We'll take a look at cleaning this up. However, what if we do not know the encoding of the file? Should the file be output in it's original binary form? Or do you know if it's invalid for a CSV to contain both types?

**Comment by craigez** [comment]
This should be resolved in #7
